terracumber_config/tf_files/SUSEManager-4.1-qam.tf

```diff
@@ -866,6 +864,135 @@ module "ubuntu1604-sshminion" {
   ssh_key_path = "./salt/controller/id_rsa.pub"
 }
 
+module "sles11sp4-buildhost" {
+  providers = {
+    libvirt = libvirt.giediprime
+  }
+  source             = "./modules/minion"
+  base_configuration = module.base_old_sle.configuration
+  product_version    = "4.1-released"
+  name               = "buildhost-sles11sp4"
+  image              = "sles11sp4"
+  provider_settings = {
+    mac                = "aa:b2:92:6a:52:83"
```

Comment:
Two of the MACs are already assigned:
```
suma-qam-41-min-sles11sp4: aa:b2:92:6a:52:83
suma-qam-41-min-sles12sp4: aa:b2:92:9a:94:c9
```

Shouldn't we add new uniq MACs for these added VMs?

Reply:
Good catch, I already prepared new mac addresses to be used in the second PR for infrastructure, but I forgot to paste them here. Thanks a lot, I will fix it asap :-)